https://bugs.kde.org/show_bug.cgi?id=381087#c4

--- origsrc/ksudoku-17.04.3/src/gui/org.kde.ksudoku.desktop	2017-06-12 19:44:11.000000000 -0500
+++ src/ksudoku-17.04.3/src/gui/org.kde.ksudoku.desktop	2017-07-21 01:32:29.721847100 -0500
@@ -50,7 +50,7 @@ Name[wa]=KSudoku
 Name[x-test]=xxKSudokuxx
 Name[zh_CN]=KSudoku
 Name[zh_TW]=KSudoku
-Exec=ksudoku %i -qwindowtitle %c
+Exec=ksudoku %i -caption %c
 Icon=ksudoku
 Type=Application
 X-DocPath=ksudoku/index.html
